src/main/java/org/takes/rq/RqWithHeader.java

```diff
@@ -78,4 +78,15 @@ public InputStream body() throws IOException {
         );
     }
 
+    /**
+     * Add additional header.
+     * @param name Header name
+     * @param value Header value
+     * @return Request with extra header
+     */
+    public RqWithHeader withAdditionalHeader(final CharSequence name,
```

Comment:
I don't think it's a good idea. We don't have this methods anywhere. We're always using constructors. Let's follow this convention. You can create `RqWithHeaders`, similar to `RsWithHeaders`, if you want to make that cascading object constructing look "shorter"